packages/lab/src/stepped-tracker/Step.css

```diff
@@ -0,0 +1,59 @@
+.saltStep {
+  /* Copy of size calculations of <Icon /> */
+  --icon-base-size: var(--salt-size-icon, 12px);
+  --icon-size-multiplier: var(--saltIcon-size-multiplier, 1.5);
+  --icon-size: calc(var(--icon-base-size) * var(--icon-size-multiplier));
+}
+
+.saltSteppedTracker-horizontal .saltStep {
+  position: relative;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, min-content);
+  grid-template-areas:
+    "icon"
+    "label"
+    "description";
+  justify-items: center;
+  align-items: center;
+  text-align: center;
+  flex: 1;
+  padding: 0 var(--salt-spacing-25);
+}
+
+.saltSteppedTracker-vertical .saltStep {
+  display: grid;
+  grid-template-columns: var(--icon-size) 1fr min-content;
+  grid-template-areas:
+    "icon      label       expand"
+    "connector description ."
+    "SteppedTracker   SteppedTracker     SteppedTracker";
```

Comment:
Some is short hand, some is full component name?

Reply:
yeah, this came from the renaming process, will switch it back

Reply:
Done!